scaffold-package-command.php

```diff
@@ -4,7 +4,10 @@
 	return;
 }
 
-require_once __DIR__ . '/inc/ScaffoldPackageCommand.php';
+$autoload = dirname( __FILE__ ) . '/vendor/autoload.php';
+if ( file_exists( $autoload ) && ! class_exists( 'WP_CLI\ScaffoldPackageCommand' ) ) {
```

Comment:
Why do you only load the autoloader if this class does not exist?

This goes against common expectations and might lead to obscure bugs down the road where the code relies on the autoloader but the autoloader might not be loaded under certain conditions.